src/PanelTraits/RequiredFields.php

```diff
@@ -26,7 +26,10 @@ public function setRequiredFields($class, $operation)
 
         if (count($rules)) {
             foreach ($rules as $key => $rule) {
-                if (strpos($rule, 'required') !== false) {
+                if (
```

Comment:
https://secure.php.net/manual/en/function.strpos.php

```

Return Values ¶

Returns the position of where the needle exists relative to the beginning of the haystack string (independent of offset). Also note that string positions start at 0, and not 1.

Returns FALSE if the needle was not found.
Warning

This function may return Boolean FALSE, but may also return a non-Boolean value which evaluates to FALSE. Please read the section on Booleans for more information. Use the === operator for testing the return value of this function.
```

Reply:
Thanks for the feedback. Please note, that we want to evaluate, that array_search does find the string 'required' in the array. So the supposed return value is anything unequal false. You are right, that we need to perform a type sensitive comparison. That is why I used `!==` instead of a simple `!=`.
In other words: we need the inverse of `===`. And that would be `!==`, which is what I did in my first commit.

Please correct me, if i got your feedback wrong. I respond from my mobile and might have missed a thing.

Reply:
Totally agree with you here @mark-win ! I think ```!==``` will do just fine.